addon/adapters/drf.js

```diff
@@ -111,5 +96,21 @@ export default DS.RESTAdapter.extend({
     } else {
       return error;
     }
+  },
+
+  /**
+   * This is used by RESTAdapter.groupRecordsForFindMany.
+   *
+   * The original implementation is a complex stripping of the id from
+   * the URL, which can be dramatically simplified by just returning
+   * the base URL for the type.
+   *
+   * @method _stripIDFromURL
+   * @param {DS.Store} store
+   * @param {DS.Model} record
+   * @return {String} url
+   */
+  _stripIDFromURL: function(store, record) {
+    return this.buildURL(record.constructor.typeKey);
   }
```

Comment:
Could you update the comment to add a note that this is required because of the trailing slashes issue? The motivation for this change isn't clear at the moment.